inc/admin.php

```diff
@@ -48,6 +49,17 @@ public function __construct() {
 		}
 	}
 
+	/**
+	 * Init no_script setup value based on whether the user is connected or not.
+	 */
+	public function init_no_script() {
+		if ( $this->settings->is_connected() ) {
+			$raw_settings = $this->settings->get_raw_settings();
```

Comment:
@GrigoreMihai looking at this it seems that is always executing, no matter if the user is new or not. Considering that the user is new, once they connect the plugin, will go into this logic, no? 

Have you tested this logic?

Reply:
yup, for new users that are connecting the plugin it will be off, it my local testing is_connected is returning false at the begining and the default schema will be used before this check will go through , after the following calls will have the option setup from the default schema and it will be off unless the users changes this.

Reply:
@GrigoreMihai ok, I have looked over the code, and I don't see any place where we set up the option from the default schema; we only use it when getting the values, but we are not setting them. Since in your logic, you use raw values, the schema is ignored, at least, this is how I see it. 

Maybe I'm missing something, but I guess we can update the testing instructions to address this case.

Reply:
Yup it was intended to skip the schema to cover the case when the user updated the setting otherwise we get the default `disabled` and we have no way to know if the user set it or not. 

I updated the test comment. If it's ok with you I can move it to testing?

Reply:
@GrigoreMihai for me, the PR testing instructions look the same and not much helpful for the QA https://vertis.d.pr/i/dbWnH4

Reply:
For testing I used a separate comment here: https://github.com/Codeinwp/optimole-wp/pull/521#issuecomment-1513424105 , https://vertis.d.pr/i/YKhixR , the one you mentioned was only for detailing the code logic I thought it is better to not edit that , ofc I can also change that if do not think it's a good idea ?

Reply:
@GrigoreMihai, I noticed that the PR description could be more relevant to the context and should not contain outdated information. Since this is the third time I'm mentioning this, I kindly request you to follow the PR template closely and provide detailed testing instructions for the QA team in the description. You can refer to PRs in the Otter/Neve repositories as excellent examples. Thanks!